src/views/v2/Bridge/Routes/index.tsx

```diff
@@ -118,26 +124,46 @@ const Routes = ({ ...props }: Props) => {
           }
         }
       }
+
+      // Add manual routes that aren't already in topRoutes
+      for (const manualRoute of manualRoutes) {
```

Comment:
O(n2) since loop inside loop.
Can I think change it to `[...new Set([...topRoutes, ...manualRoutes])]` to make it O(n)